path/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Deno
```

Comment:
Copyright 2018 the Deno authors.

Please move this file to the root.

Reply:
I am done.